### PR TITLE
feat(docs): adds archive docs / examples

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

File: etc/hub.api.md [hunk 1]
```diff
@@ -69,10 +69,30 @@ export type APISig = {
...
+};
+
 export { ArchiveInfoResponse }
 
 export { ArchiveResponse }
 
+// @public (undocumented)
+export type ArchiveStatus = {
+    key: string;
+    status: number;
```
Comment (andrewxhill) [inline etc/hub.api.md:92]: where can i get the status number mapping @asutula ?

Comment (asutula) [reply etc/hub.api.md:92]: Its in the generated proto code I guess you're generating these `ArchiveStatus` objects from. Looks like:
```
export namespace ArchiveStatusResponse {
  export type AsObject = {
    key: string,
    status: ArchiveStatusResponse.StatusMap[keyof ArchiveStatusResponse.StatusMap],
    failedMsg: string,
  }

  export interface StatusMap {
    STATUS_UNSPECIFIED: 0;
    STATUS_EXECUTING: 1;
    STATUS_FAILED: 2;
    STATUS_DONE: 3;
    STATUS_CANCELED: 4;
  }

  export const Status: StatusMap;
}```

Comment (andrewxhill) [reply etc/hub.api.md:92]: oop, missed that.

**Comment by andrewxhill** [comment]
k, updated that

**Comment by andrewxhill** [comment]
gtg

**Pull request merged**